src/Methods/Address/GetUnspents.ts

```diff
@@ -90,7 +92,7 @@ export default method({
       const [inscriptions, ordinals, runeBalances] = await Promise.all([
```

Comment:
Rename `runeBalances` to `hasRunes` for clarity and consistency with `GetSpendables`